.github/workflows/autobump.yml

```diff
@@ -1,65 +0,0 @@
-name: autobump
-
-on:
-  push:
-    branches: [ "main" ]
-
-jobs:
-  version_bump:
-    runs-on: ubuntu-latest
-    outputs:
-      version: ${{ steps.bump.outputs.tag }}
-    steps:
-      - uses: actions/checkout@v3
-
-      - name: Bump version and push tag
-        id: bump
-        uses: anothrNick/github-tag-action@1.64.0
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          WITH_V: true
-          DEFAULT_BUMP: patch
-
-  build:
-    needs: version_bump
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        platform: [linux/amd64, linux/386, linux/arm/v5, linux/arm/v7, linux/arm64/v8, linux/ppc64le, linux/s390x]
-    steps:
-      - uses: actions/checkout@v3
-
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
-
-      - name: Login to Docker Hub
-        uses: docker/login-action@v2
-        with:
-          username: ${{ secrets.DOCKER_HUB_USERNAME }}
-          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
-
-      - name: Docker meta
-        id: meta
-        uses: docker/metadata-action@v4
-        with:
-          images: |
-            datapreservationprogram/singularity
-          tags: |
-            type=schedule
-            type=ref,event=branch
-            type=ref,event=pr
-            type=semver,pattern={{raw}},value=${{ needs.version_bump.outputs.version }}
-            type=raw,value=latest,enable={{is_default_branch}}
-
-      - name: Build and push
```

Comment:
This will also stop container publication to dockerhub, the container registry referenced in Singularity docs. 

I recommend either keeping this bit or update docs to point to ghcr.io now that CI is building/publishing to it.

Reply:
I will move it to ghcr.io.
I'll make the doc update in a different PR, or through gitbook directly.

Reply:
I don't know how widely used the docker hub container registry for singularity is. but the change seems worthy of some notice if that's going to break clients / silently stop having the latest.